spec.html

```diff
@@ -28717,130 +28485,76 @@ <h1>Math.round ( _x_ )</h1>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If _x_ is *-&infin;*, the result is *-&infin;*.
-          </li>
-        </ul>
+        <p>Returns the hyperbolic sine of _x_.</p>
+        <p>When the `Math.sinh` method is called with argument _x_, the following steps are taken:</p>
+        <emu-alg>
+            1. Let _n_ be ? ToNumber(_x_).
+            1. If _n_ is *NaN*, _n_ is *-0*, _n_ is *+0*, _n_ is *+&infin;*, or _n_ is *-&infin;*, return _n_.
+            1. Return an implementation-approximated value representing the result of the hyperbolic sine of _n_.
+        </emu-alg>
         <emu-note>
           <p>The value of `Math.sinh(x)` is the same as the value of `(Math.exp(x) - Math.exp(-x)) / 2`.</p>
         </emu-note>
       </emu-clause>
 
       <emu-clause id="sec-math.sqrt">
         <h1>Math.sqrt ( _x_ )</h1>
-        <p>Returns an implementation-approximated value representing the result of the square root of _x_.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is less than 0, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;*, the result is *+&infin;*.
-          </li>
-        </ul>
+        <p>Returns the square root of _x_.</p>
+        <p>When the `Math.sqrt` method is called with argument _x_, the following steps are taken:</p>
+        <emu-alg>
+            1. Let _n_ be ? ToNumber(_x_).
+            1. If _n_ is *NaN*, _n_ is *-0*, _n_ is *+0*, or _n_ is *+&infin;*, return _n_.
+            1. If _n_ &lt; 0, return *NaN*.
+            1. Return an implementation-approximated value representing the result of the square root of _n_.
+        </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-math.tan">
         <h1>Math.tan ( _x_ )</h1>
-        <p>Returns an implementation-approximated value representing the result of the tangent of _x_. The argument is expressed in radians.</p>
-        <ul>
-          <li>
-            If _x_ is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If _x_ is *+0*, the result is *+0*.
-          </li>
-          <li>
-            If _x_ is *-0*, the result is *-0*.
-          </li>
-          <li>
-            If _x_ is *+&infin;* or *-&infin;*, the result is *NaN*.
-          </li>
-        </ul>
+        <p>Returns the tangent of _x_. The argument is expressed in radians.</p>
+        <p>When the `Math.tan` method is called with argument _x_, the following steps are taken:</p>
+        <emu-alg>
+            1. Let _n_ be ? ToNumber(_x_).
+            1. If _n_ is *NaN*, _n_ is *-0*, or _n_ is *+0*, return _n_.
+            1. If _n_ is *-&infin;*, or _n_ is *+&infin;*, return *NaN*.
```

Comment:
```suggestion
            1. If _n_ is *+&infin;* or _n_ is *-&infin;*, return *NaN*.
```

(note also the stray comma)